common/src/thrift/common.thrift

```diff
@@ -44,3 +44,13 @@ struct NetAddress {
   3: i32 dataPort
 }
 
+service TachyonService {
+
+  /*
+   * Returns the version of the master service.
+   *
+   * NOTE: The version should be updated every time a backwards incompatible API change occurs.
+   */
+  i64 getServiceVersion()
+
```

Comment:
It looks like our other services use a convention of blank line at start, no blank line at end.